WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -0,0 +1,167 @@
+package org.wordpress.android.ui.accounts
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.View
+import android.widget.TextView
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.AccountStore
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.support.createNewTicket
+import org.wordpress.android.support.showAllTickets
+import org.wordpress.android.support.showSupportEmailInputDialog
+import org.wordpress.android.support.showZendeskHelpCenter
+import org.wordpress.android.ui.ActivityId
+import org.wordpress.android.ui.AppLogViewerActivity
+import org.wordpress.android.ui.prefs.AppPrefs
+import org.wordpress.android.util.LocaleManager
+import java.util.ArrayList
+import javax.inject.Inject
+
+class HelpActivity : AppCompatActivity() {
+    @Inject lateinit var accountStore: AccountStore
+    @Inject lateinit var siteStore: SiteStore
+
+    private val originFromExtras by lazy {
+        (intent.extras?.get(HelpActivity.ORIGIN_KEY) as Origin?) ?: Origin.UNKNOWN
+    }
+    private val extraTagsFromExtras by lazy {
+        intent.extras?.getStringArrayList(HelpActivity.EXTRA_TAGS_KEY)
+    }
+    private val selectedSiteFromExtras by lazy {
+        intent.extras?.get(WordPress.SITE) as SiteModel?
+    }
+
+    override fun attachBaseContext(newBase: Context) {
+        super.attachBaseContext(LocaleManager.setLocale(newBase))
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        (application as WordPress).component().inject(this)
+
+        setContentView(R.layout.help_activity)
+
+        val actionBar = supportActionBar
+        if (actionBar != null) {
+            actionBar.setHomeButtonEnabled(true)
+            actionBar.setDisplayHomeAsUpEnabled(true)
+            actionBar.elevation = 0f // remove shadow
+        }
+
+        val contactUsButton = findViewById<TextView>(R.id.contact_us_button)
+        contactUsButton.setOnClickListener { createNewZendeskTicket() }
```

Comment:
You could use Android Kotlin extensions here and avoid inflating the views manually by using `import kotlinx.android.synthetic.main.help_activity.*`. Just note that you'll have to rename the view ids in the layout XML if you want camel-case variable names.

Reply:
That sounds like a good idea. Do we already use that pattern elsewhere in the app? I don't want to do something inconsistent as part of Zendesk work.